crates/modelardb_common/src/arguments.rs

```diff
@@ -205,6 +208,43 @@ pub async fn parse_azure_blob_storage_arguments(
     Ok(azure_blob_storage)
 }
 
+/// Parse the arguments in `data` and return a [`PgPool`] of connections to a PostgreSQL database
```

Comment:
The input to this function is from the manager and is generated by a function on the side of the manager so I do not think it is necessary to provide a description of exactly how `data` is structured. If it is used more in the future it might be relevant to change this later. 

By remaining we mean that when we extract the arguments used to connect to the PostgreSQL database, we return the rest of the bytes in `data`. This is because `data` contains everything that is necessary to connect to both the metadata database and remote object store.